Formula/balena-cli.rb

```diff
@@ -32,9 +30,8 @@ class BalenaCli < Formula
 
   def install
     ENV.deparallelize
-    ENV.prepend_path "PATH", Formula["python@3.10"].opt_libexec/"bin"
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    (bin/"balena").write_env_script libexec/"bin/balena", PATH: "#{Formula["node@14"].opt_bin}:$PATH"
+    system Formula["node@14"].opt_bin/"npm", "install", *Language::Node.std_npm_install_args(libexec)
```

Comment:
`setup_npm_environment` automatically prepends `node` to `PATH`, which causes issues when using `system "npm"` on systems with `node` installed already. 
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/language/node.rb#L48

Using full path to `npm` to avoid source build failures.